_1327/main/templates/searched_list.html

```diff
@@ -0,0 +1,55 @@
+{% extends 'base_with_sidebar.html' %}
+{% load i18n %}
+
+{% block title %}
+	{% blocktrans count counter=2 %}Documents{% plural %}Documents{% endblocktrans %}
+{% endblock %}
+
+{% block sidebar %}
+	<div class="toc hidden-print">
+		<ul>
+			{% for type, documents in searched_documents %}
+				<li><a href="#type{{ type }}">{{ type }}</a></li>
+			{% endfor %}
+		</ul>
+	</div>
+{% endblock %}
+
+{% block content %}
+	{% for type, documents in searched_documents %}
+		<h3 id="type{{ type }}">{{ type }}</h3>
+		<table class="table table-striped">
+			{% for document, lines in documents %}
+				<tr>
+					<td style="width: 55%;">
```

Comment:
could you put the style annotations into a `.less` file? For instance `1327.less`. You can do so by adding a new class.